### ISSUE TITLE
Out expand command definition for tenant:run

### ISSUE THREAD

**Issue by gisostallenberg** [archtechx/tenancy#686]
#### Description
Just add the arguments and options of the target sub command as allowed options so a command can be copied from existing docs.
Example:
Current:
```
php artisan tenants:run --tenants=xxxxx route:list --option="compact"
```
New:
```
php artisan tenants:run --tenants=xxxxx route:list --compact
```

#### Why this should be added
This allows to simply copy any existing command mentioned in docs and elsewhere.

#### 
I tested this with the following code and am willing to provide a PR.

```php
<?php

declare(strict_types=1);

namespace Stancl\Tenancy\Commands;

use Illuminate\Console\Command;
use Illuminate\Console\OutputStyle;
use Symfony\Component\Console\Input\ArgvInput;
use Symfony\Component\Console\Input\InputDefinition;
use Symfony\Component\Console\Input\InputInterface;
use Symfony\Component\Console\Output\OutputInterface;

class Run extends Command
{
    /**
     * The console command description.
     *
     * @var string
     */
    protected $description = 'Run a command for tenant(s)';

    /**
     * The name and signature of the console command.
     *
     * @var string
     */
    protected $signature = "tenants:run {commandname : The command's name.}
                            {--tenants=* : The tenant(s) to run the command for. Default: all}
                            {--argument=* : The arguments to pass to the command. Default: none}
                            {--option=* : The options to pass to the command. Default: none}";

    /**
     * The sub arguments for this run
     *
     * @var array
     */
    private $subArguments = [];

    /**
     * Run the console command.
     *
     * @param  \Symfony\Component\Console\Input\InputInterface  $input
     * @param  \Symfony\Component\Console\Output\OutputInterface  $output
     * @return int
     */
    public function run(InputInterface $input, OutputInterface $output)
    {

        $this->subArguments = $this->parseArgv();
        if (empty($this->subArguments)) {
            return parent::run($input, $output);
        }

        $subCommand = $this->getApplication()->find($this->subArguments[0]);
        $this->getDefinition()->addArguments($subCommand->getDefinition()->getArguments());
        $this->getDefinition()->addOptions($subCommand->getDefinition()->getOptions());

        return parent::run($input, $output);
    }

    private function parseArgv()
    {
        $argv = $_SERVER['argv'] ?? [];
        array_shift($argv);
        $subCommandFound = false;
        $thisCommandFound = false;
        $arguments = [];
        foreach ($argv as $arg) {
            if ($arg === 'tenants:run') {
                $thisCommandFound = true;
                continue;
            }
            if ($thisCommandFound && $subCommandFound === false && strpos($arg, '-') === 0) {
                continue; // still options of tenant command
            }
            if ($thisCommandFound && $subCommandFound === false) {
                $arguments[] = $arg;
                $subCommandFound = true;
                continue;
            }
            if ($thisCommandFound && $subCommandFound !== false) {
                $arguments[] = $arg;
            }
        }

        return $arguments;
    }

    /**
     * Execute the console command.
     *
     * @return mixed
     */
    public function handle()
    {
        $subArguments = $this->subArguments;
        tenancy()->runForMultiple($this->option('tenants'), function ($tenant) use ($subArguments) {
            $this->line("Tenant: {$tenant['id']}");
            tenancy()->initialize($tenant);

            $subCommand = $this->getApplication()->find($this->subArguments[0]);
            $input = new ArgvInput($this->subArguments, $subCommand->getDefinition());
            $subCommand->run($input, $this->output);
        });
    }
}
```

**Comment by gisostallenberg** [comment]
@stancl I see you added the v4 label. Is this a version in near future? Because I could also make a BC solution...

**Comment by stancl** [comment]
Yes, see the pinned issue. I'll make all of these changes at once.

**Comment by gisostallenberg** [comment]
👍🏽  Thanks. Also for the nice package you provide!

**Comment by stancl** [comment]
Implemented in #912
